docker-compose.yaml

```diff
@@ -1,32 +1,32 @@
-version: '2'
+version: '3.2'
 
 services:
-  service_php:
+  php:
```

Comment:
As already said, I renamed the name of the services, service_ seems redundant to me.